src/main/java/com/redhat/coolstore/service/CatalogService.java

```diff
@@ -9,7 +9,7 @@
 import javax.persistence.criteria.CriteriaQuery;
 import javax.persistence.criteria.Root;
 
-import javax.ejb.Stateless;
+import jakarta.ejb.Stateless;
```

Comment:
This is just wrong, and should be causing a maven compiler issue, but did not see them.